Assets/Scripts/AnimateNPC.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimateNPC : MonoBehaviour
+{
+    /// <summary>
+    /// The NPC's Animator component
+    /// </summary>
+    private Animator animator;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        animator = GetComponent<Animator>();
+    }
+
+    /// <summary>
+    /// Sets the NPC's animation direction to face the player initiating the conversation
+    /// </summary>
+    /// <param name="playerTransform">The location of the player.</param>
+    public void SetAnimationDirectionOnConversationStart(Transform playerTransform)
+    {
+        Vector2 playerDirection = GetOneDimensionalDirectionalVector(playerTransform.position);
+        if (playerDirection == Vector2.down)
+            animator.SetTrigger("Move Down");
+        else if (playerDirection == Vector2.up)
+            animator.SetTrigger("Move Up");
+        else if (playerDirection == Vector2.left)
+            animator.SetTrigger("Move Left");
+        else if (playerDirection == Vector2.right)
+            animator.SetTrigger("Move Right");
+
+    }
+
+    /// <summary>
+    /// Creates a normalized directional vector from the distance between the NPC and player
```

Comment:
just explain this a bit more clearly if possible

Reply:
consider renaming. My understanding is that this returns a 1 unit vector 2 pointed from the npc to the player. So figure out how to say that / name it.